index.html

```diff
@@ -0,0 +1,311 @@
+      <div class="section_header">How It Works</div>
+      <figure class="row flex_wrap workflow_row">
+        <figure class="workflow_figure">
+          <div class="workflow_image"></div>
+          <figcaption>input <code>markdown</code></figcaption>
+        </figure>
+        <div class="workflow_arrow">
+          <i class="fas fa-long-arrow-alt-right fa-lg"></i>
+        </div>
+        <figure class="workflow_figure">
+          <div class="workflow_image"></div>
+          <figcaption>
+            process w/ Manubot tools and <a href="https://pandoc.org/">Pandoc</a>
+          </figcaption>
+        </figure>
+        <div class="workflow_arrow">
+          <i class="fas fa-long-arrow-alt-right fa-lg"></i>
+        </div>
+        <figure class="workflow_figure">
+          <div class="workflow_image"></div>
+          <figcaption>
+            output <code>.html</code>, <code>.pdf</code>, <code>.docx</code>
+          </figcaption>
+        </figure>
+        <div class="workflow_arrow">
+          <i class="fas fa-long-arrow-alt-right fa-lg"></i>
+        </div>
+        <figure class="workflow_figure">
+          <div class="workflow_image"></div>
+          <figcaption>test, deploy to host</figcaption>
+        </figure>
+      </div>
+    </section>
+    <section>
+      <div class="section_header">The Best Parts</div>
+      <div class="row flex_wrap_reverse">
+        <div class="col_text">
+          <div class="col_header">
+            <i class="fas fa-atlas fa-lg"></i> Open, collaborative, versioned
+          </div>
+          <div>
+            Author your manuscript the open-source way.
+            Let anyone propose changes.
+            Iteratively review, discuss, and refine with collaborators.
+            Coordinate teams of any size with ease.
+            Keep track of versions, to-dos, revisions, and who wrote what and when.
+            All in a <a href="https://github.com/">central location</a>.
+          </div>
+        </div>
+        <div class="col_image"></div>
+      </div>
+      <div class="row flex_wrap">
+        <div class="col_image"></div>
+        <div class="col_text">
+          <div class="col_header">
+            <i class="fas fa-magic fa-lg"></i> Automated and continuous
+          </div>
+          <div>
+            Never use a reference manager again.
+            Provide a DOI, ISBN, PubMed ID, URL, etc., and let Manubot generate and number citations for you.
+            Choose from <a href="https://www.zotero.org/styles">thousands of existing citation styles</a>.
+            See your manuscript update right away as you make changes.
+            Test and deploy automatically with <a href="https://travis-ci.org/">continuous integration</a>.
+          </div>
+        </div>
+      </div>
+      <div class="row flex_wrap_reverse">
+        <div class="col_text">
+          <div class="col_header">
+            <i class="fas fa-hand-point-up fa-lg"></i> Beautiful and interactive
```

Comment:
In your HTML preview I noticed that this icon looks like it is pointing to the `Automated and continuous` image above, not the image to the right.  Maybe use the font awesome `comments` icon for "public annotations" or `mouse-pointer` for "interactive"?

Reply:
I think that hand icon is pretty firmly associated with a mouse cursor rather than pointing in a direction, nowadays. We could use `mouse-pointer`, though I prefer the hand to indicate that we've  thought about mobile (touch devices) too, which is hugely important and something that a lot of the other tools don't support (like lens viewer).

I understand `comments` w/ the annotations plugin, but that's just one of many. I was thinking of having something more vague.

What about something more vague like `eye` `star` or `bolt`.